addons/mail/static/src/core/common/composer.xml

```diff
@@ -237,4 +237,26 @@
             <t t-esc="option.emoji.shortcodes.join(' ')"/>
         </em>
     </t>
+
+    <t t-name="mail.Suggestion.Partner">
+        <a t-att-href="href" t-att-data-oe-id="partnerId" data-oe-model="res.partner" target="_blank" contenteditable="false" draggable="false"
```

Comment:
Or guideline for attribute order is: class, href, target, content-editable, data- stuff